src/Cake.Testing/FakeLog.cs

```diff
@@ -37,6 +39,18 @@ public Verbosity Verbosity
         }
 
         /// <summary>
+        /// Aggregates all current log entries message
+        /// </summary>
+        /// <returns>Log messages as <see cref="System.String"/></returns>
+        public string AggregateLogMessages()
```

Comment:
Can we change this to an extension method in the Cake.Common.Tests assembly instead since Cake.Testing is a public assembly?